mesonbuild/coredata.py

```diff
@@ -974,6 +974,8 @@ def set_options(self, options: T.Dict[OptionKey, T.Any], subproject: str = '', f
         return dirty
 
     def set_default_options(self, default_options: T.MutableMapping[OptionKey, str], subproject: str, env: 'Environment') -> None:
+        from .compilers import base_options
```

Comment:
@bruchar1 this commit broke subprojects for me:

    subprojects/mysubproj/meson.build:1:0: ERROR: In subproject mysubproj: Unknown options: "mysubproj:b_ndebug"